### DISCUSSION TITLE
preset_modes modifications/custom possible?

### DISCUSSION THREAD

**Discussion by turbolooser** [revk/xESP32-Faikout#444] [General]
Would anyone have an idea on how to modify the existing preset_modes: none, eco, boost, home ? Or maybe add custom preset_modes? 

Any help is appreciated.
Thanks

**Comment by revk** [comment]
Try the latest beta as a load of switches have been added to HA which may do what you needs.

**Reply by turbolooser** [reply]
Loving those new switches ... thanks for that. Still I am looking for something like a prest_mode I could edit the existing ones like "home" or "eco". Where/How could I edit the parameters configured ion these presets?

**Reply by revk** [reply]
They are party of the code - you could change your HA config, which may be possible for some things, but a change to teh code would be needed to understand the `preset` value you are sending/seeing.

What is the actual change you want, and why?

**Reply by turbolooser** [reply]
I was just thinking If I could create my own preset per indoor unit (having 3) I would be able to just set a schedule for all units at once including a custom preset e.g.

summer(livingroom) - 20 temp, fan=comfort
summer(kitchen) - 22 temp, fan=h+v
summer(bedroom) - 18 temp, fan=comfort

Then in the scheduler I could say turn on all indoor units with preset=summer and all units would have their specifi settings.
Doing this now I need 3 schedules (1 for each indoor unit) rather than one. Including some additional rules this becomes kinda messy for summertime and wintertime and being present and not.

makes sense?

**Reply by revk** [reply]
I am not sure that is how the `preset` is really envisaged. Surely HA has a way of setting all of that in one go itself, some internal sort of `preset`. Adding configurable *presets* would obviously be possible.

Not out of the question, obviously.

Indeed, the existing `preset` probably should be removed as we have *switches* that cover it all now.
